app/components/util/how-do-you-feel.jsx

```diff
@@ -0,0 +1,64 @@
+//https://github.com/EnCiv/civil-pursuit/issues/62
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import { Button } from '../button'
+
+const responseOptions = ['Awesome!', 'Just Okay', 'Unsatisfied']
+export default function HowDoYouFeel(props) {
+    const { disabled = false, className, onDone, title = '', ...otherProps } = props
+    const styleClasses = rankingStyleClasses(props)
+    const onSelection = e => {
+        onDone({ valid: true, value: e.target.value })
+    }
+    return (
+        <>
```

Comment:
need to change this to a `<div className={className} {...otherProps} />` so that a parent component can apply those things to this component.